docs_src/src/pages/documentation/api_reference/multithreading.mdx

```diff
@@ -0,0 +1,94 @@
+export const description =
+  'On this page, we’ll dive into the different conversation endpoints you can use to manage conversations programmatically.'
+
+## Variables in Multithreading
+
+Batman wondered about the behavour of variables in a Robyn multiprocessing environment.
+
+
+<Row>
+<Col>
+
+Any variable used in a multiprocessing environment is shared across multiple processes.
+
+Whilst using multithreading in Robyn, the variables are not protected from multiple threads access by default.
+
+We need the value to be protected within the process, while accessing the count from different threads.
```

Comment:
This is still not enough imo 😓 

First, we should explain - `robyn can be run in a multiprocess manner. which means that the handlers can be dispatched to multiprocesses`

then explain

```then explain - should you wish to access a state shared across processes, you need to protect it with `mp.value` ```

then finally give an example of multi threading

Reply:
hi, pleast take a look now.